java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -298,6 +302,9 @@ private static Object[] listCommands(BaseTransaction<?> transaction) {
                     4L, // linsert(listKey3, AFTER, value2, value2)
                     new String[] {listKey3, value3}, // blpop(new String[] { listKey3 }, 0.01)
                     new String[] {listKey3, value1}, // brpop(new String[] { listKey3 }, 0.01)
+                    2L, // lpush(listKey5, new String[] {value2, value3})
+                    OK, // lset(listKey5, 0, value1)
+                    new String[] {value1, value2} // lrange(listKey5, 0, -1)
```

Comment:
```suggestion
                    new String[] {value1, value3} // lrange(listKey5, 0, -1)
```
aren't we changing the value at index 0 only?

Reply:
since lpush is LIFO the list looks like `{value3, value2}` after first being pushed to, and after setting index 0 it becomes `{value1, value2}`